### ISSUE TITLE
A html span tag is printed when an extra pair of brackets is used in the bib file.

### ISSUE THREAD

**Issue by whcjimmy** [zepinglee/citeproc-lua#61]
**Describe the bug**
When appying an extra pair of brackets in the bib file, the parsing result is incorrect. As figured, a html span tag is printed. 

**Additional information**
- TeX distribution: MacTex 2023.0314
- Package `citation-style-language` version: 0.3.0
- LaTeX engine: `lualatex`

**To Reproduce**
```tex
\RequirePackage{filecontents}

\begin{filecontents*}{test.bib}
@misc{BAMF_2022, 
    author = {{Bundesamt für Migration und Flüchtlinge}},
    title = {Curriculum für einen bundesweiten Orientierungskurs},
    url={https://www.bamf.de/SharedDocs/Anlagen/DE/Integration/Integrationskurse/Kurstraeger/KonzepteLeitfaeden/curriculum-orientierungskurs-pdf.html}, 
    year={2017}, 
    urldate = {2024-04-26},
}
\end{filecontents*}

\documentclass{article}         

% Customize Bibliography Style
\usepackage{citation-style-language}
\cslsetup{style = apa}
\addbibresource{test.bib}  % or example.bib or example.yaml

\begin{document}

\nocite{*}
\printbibliography

\end{document}
```

**Screenshots**
<img width="624" alt="image" src="https://github.com/zepinglee/citeproc-lua/assets/4002046/5a91502d-bf95-4672-9693-a99b9bdb22a9">

**Comment by zepinglee** [comment]
This bug has been resolved in the latest version (v0.4.9). You may consider upgrading the MacTeX/TeX Live distribution, or installing the package from the repository (i.e., checking out the repo and run `l3build install` which installs it to `"$(kpsewhich -var-value TEXMFHOME)"`).

<img width="606" alt="Screenshot 2024-04-26 at 20 17 32" src="https://github.com/zepinglee/citeproc-lua/assets/12290822/53a24b89-471c-4d9c-95b3-74514a8ee184">

**Comment by whcjimmy** [comment]
Thank you for your help. I got what I need!
